docs/src/components/ReactExample/components/Board.tsx

```diff
@@ -96,6 +104,18 @@ const Board = ({
               Playground
             </ButtonLink>
           )}
+          {variants.length > 0 && (
+            <Popover content={<Variants exampleVariants={variants} onChange={onChangeVariant} />}>
```

Comment:
It would be better if the current variant was already selected when opening this, would this be hard to do?